src/coderdojochi/views/general.py

```diff
@@ -89,6 +89,22 @@ def upcoming_classes(self):
         return upcoming_classes[:3]
 
 
+class AboutView(TemplateView):
+    template_name = "about.html"
+    
+    @cached_property
+    def students_served_count(self):
+        students_served_count = Order.objects.exclude(check_in=None).count()
+        
+        return students_served_count 
+
+    def get_context_data(self, **kwargs):
```

Comment:
You don't need `get_context_data`, but you will need to create a method to return the `mentor_count`.

Reply:
@ali I encouraged her to use `get_context_data` to assign the internal getters to the context explicitly. 

Context: Open to discuss but imo leveraging methods and properties directly off the view in the templates is a bad idea because then you can't pass any arguments to them (aside from self) so youll end up using `get_context_data` in that case anyway then it's not explicit what is going to the template, some direct from the view, some from the context, etc

Reply:
@bkellgren The reason for one-off methods like the one I proposed is for that reason alone, for data that doesn't need other arguments.

I think polluting the `get_context_data` method get ugly, error prone, and really hard to read and maintain.

@bkellgren or @pjsier  Do we have a case anywhere that we need to pass in a variable to a `get` method?

Reply:
@karbassi  I'm not sure we add additional arguments to any yet today but its a common django pattern so just trying to future proof.  For the record, accessing the view directly in the templates is an anti-pattern as far as class-based views go i.e. not recommended in the docs.

If we do go with directly accessing the view in the templates then we need to be sure to use @cached_property every single time because otherwise the queries will be executed every time the item is referenced in the template.

Reply:
@karbassi I see your suggestion was already implemented but i would encourage us to explore this topic further because I really have a hard time supporting an anti-pattern to just save one `get_context_data` method per view, here are some additional reasons some SpotHero people gave for not doing this (aside from what I mentioned above about passing arguments to the properties/getters)...

- Instead of coupling your templating layer with the view code at one place, now you're doing n+1 places, this is not ideal for a number of use cases
- When you want to convert things to an API some day, all the values are cleanly documented within `get_context_data` and it can be leveraged directly by serializers

Food for thought.